etl/steps/data/garden/emissions/2023-11-23/national_contributions.meta.yml

```diff
@@ -1,330 +1,323 @@
       share_of_annual_emissions_co2_total:
-        title: Share of global CO2 emissions
-        unit: "%"
-        short_unit: "%"
-        description: |
-          Share of global CO2 emissions.
-      share_of_annual_emissions_ghg_total_co2eq:
-        title: Share of global greenhouse gas emissions
+        title: Share of global CO₂ emissions
         unit: "%"
         short_unit: "%"
-        description: |
-          Share of global greenhouse gas emissions. Greenhouse gas emissions are calculated by Our World in Data based on emissions data from Jones et al. (2023) and IPCC AR6 conversion factors. Jones et al. (2023) give methane and nitrous oxide emissions in standard metric tonnes per year. We have converted these emissions to carbon-dioxide equivalents over a 100-year timescale using a conversion factor of 273 for nitrous oxide, 29.8 for methane from fossil sources, and 27.2 for methane from agricultural and land use sources (as per the IPCC AR6 report).
+        description_short: "Measured as a percentage of the world's carbon dioxide emissions."
       share_of_annual_emissions_n2o_total:
         title: Share of global nitrous oxide emissions
         unit: "%"
         short_unit: "%"
-        description: |
-          Share of global nitrous oxide emissions. Nitrous oxide emissions are calculated by Our World in Data based on emissions data from Jones et al. (2023) and IPCC AR6 conversion factors. Jones et al. (2023) give nitrous oxide emissions in standard metric tonnes per year. We have converted these emissions to carbon-dioxide equivalents over a 100-year timescale using a conversion factor of 273 (as per the IPCC AR6 report).
+        description_short: "Measured as a percentage of the world's nitrous oxide emissions."
+      # Share of global temperature change caused by greenhouse gas emissions from each country (calculated by OWID).
       share_of_temperature_response_ghg_total:
         title: Share of contribution to global warming
         unit: "%"
         short_unit: "%"
-        description: |
-          Share of contribution to global warming (as a percentage). This measures each country's contribution to global mean surface temperature (GMST) rise from its cumulative emissions of carbon dioxide, methane and nitrous oxide. The warming effects of each gas are calculated based on cumulative CO2-equivalent emissions using the Global Warming Potential (GWP*) approach.
+        description_short: "Measured as a percentage of the world's temperature change."
+        description_key:
+          - This temperature change measures each country's contribution to global mean surface temperature (GMST) rise from its cumulative emissions of carbon dioxide, methane and nitrous oxide.
+          - The warming effects of each gas are calculated based on cumulative CO₂-equivalent emissions using the Global Warming Potential (GWP*) approach.
```

Comment:
last one I think. feel free to ignore of course but I think it'd add some value on the datapage if you give a bit more detail.